src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-pickup.html

```diff
@@ -24,32 +34,45 @@
   <script>
     (function () {
       'use strict';
+      class LancieAdminPickup extends Polymer.Element {
+        static get is() {
+          return 'lancie-admin-pickup';
+        }
 
-      Polymer({
-        is: 'lancie-admin-pickup',
-        properties: {
-          data: Object,
-          endpoint: String,
-        },
+        static get properties() {
+          return {
+            data: Object,
+            endpoint: String,
+          };
+        }
 
-        onResponse: function(e, request) {
-          if (request.succeeded) {
+        onResponse(e, request) {
+            if (!request.succeeded) {
+                this.$.getPickupError.setError("We cannot find a TicketOption with that name. Try again");
+                return;
+            }
             const data = request.response.map(person => this.mapData(person));
             this.$.table.setData(data);
-          }
-        },
+        }
 
-        mapData: function(data) {
+        mapData(data) {
             return {
                 "Ticket ID": data.id,
                 "Name": `${data.owner.profile.firstName} ${data.owner.profile.lastName}`,
                 "Email": data.owner.email,
                 "Address": `${data.owner.profile.address} ${data.owner.profile.zipcode} ${data.owner.profile.city}`,
                 "Phone number": data.owner.profile.phoneNumber,
             };
-        },
+        }
+
+        getPickup() {
```

Comment:
Correct me if I'm wrong. But isn't the naming convention in the lancie repos 'handleTapGetPickup'? If not, then you can dismiss this.